setup.py

```diff
@@ -3,13 +3,11 @@
 # You can obtain one at http://mozilla.org/MPL/2.0/.
 
 from setuptools import setup, find_packages
-import os
-import re
 
 install_requires = ['SQLALchemy', 'unittest2', 'simplejson', 'pyramid',
                     'mozsvc>=0.8', 'cornice', 'pyramid_hawkauth', 'PyMySQL',
```

Comment:
Should the `mozsvc` dependency here be changed similarly?  It's been a while since I did anything with python dependency management so I'm a bit rusty on the details.

Reply:
I wasn't sure so I ran some tests (Debian 9, Python 2.7, pip 18.1).

I added `[memcache]` to the _mozsvc_ dependency and tried installing.
If you install with pip (e.g. `pip install .` from inside the working copy) then the additional package will be picked up as expected.
If you use `python setup.py install` then _umemcache_ will still be installed but in my case I ended up running into problems with compilation of _umemcache_ - bot in my branch and _master_.

I will adjust the dependency here accordingly.